### PR TITLE
add grafana docs

### PR DESCRIPTION
**Pull request by mayberryzane**
## Summary
- adds grafana docs covering setup, visualizations, and alerts

## How did you test this change?
- click tested locally

## Are there any deployment considerations?
- no, existing `grafana-integration` route redirects to the new Grafana overview page

## Does this work require review from our design team?
- no

### CODE REVIEW

File: docs-content/general/7_integrations/grafana-integration/3_dashboards.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+slug: dashboards
+createdAt: 2021-09-17T21:48:44.000Z
+updatedAt: 2021-09-17T21:56:27.000Z
+---
+
+You can create dashboards and visualizations using a highlight.io data source. The data source exposes a query editor that lets you create custom queries to aggregate metrics derived from your sessions, errors, logs, and traces.
+
+## Getting started
+
+Once you've [installed and configured](./2_setup.md) a highlight.io data source, you're all set to create new visualizations using that data source. On any dashboard, add a new visualization and select the highlight.io data source you created earlier. ![choosing a data source](/images/docs/grafana/select-data-source.png)
```
Comment (Vadman97) [inline docs-content/general/7_integrations/grafana-integration/3_dashboards.md:12]: image doesnt work either

File: docs-content/general/7_integrations/grafana-integration/3_dashboards.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+## Query editor fields
+
+When you use the highlight.io data source, it is configurable with these fields:
+|Name|Description|
+|----|-----------|
+|`Resource`|The highlight.io resource being queried for these metrics.|
```
Comment (Vadman97) [inline docs-content/general/7_integrations/grafana-integration/3_dashboards.md:19]: would be useful to give examples here, like `sessions` vs `logs`

File: docs-content/general/7_integrations/grafana-integration/3_dashboards.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+## Examples
+
+The following are a few example queries to help you get started with your own visualizations.
+
+### Frequent HTTP requests
+
+All network activity captured by the highlight.io client is stored as a trace with `span_name` set to the request URL. To visualize how many of each of these HTTP requests are made by your frontend over time, you can filter for these types of traces with a filter `span_name=("GET *" OR "POST *")` and group on `span_name`. The `limit 10 by count` clause shows the top 10 matching results by their total count in the visualization's time range.
+
+![Frequent HTTP requests](/images/docs/grafana/frequent-http-requests.png)
```
Comment (Vadman97) [inline docs-content/general/7_integrations/grafana-integration/3_dashboards.md:35]: all the images in this page are broken

File: docs-content/general/7_integrations/grafana-integration/3_dashboards.md
Comment (Vadman97) [inline docs-content/general/7_integrations/grafana-integration/3_dashboards.md:1]: would be nice to have an example `.json` file for the dashboard here of the examples

File: docs-content/general/7_integrations/grafana-integration/4_alerts.md
Comment (Vadman97) [inline docs-content/general/7_integrations/grafana-integration/4_alerts.md:1]: some screenshots of grafana here would help with explaining the difference between the timeseries -> reduce approach vs. the single value setup.

**Review by Vadman97 — APPROVED** [review]

File: docs-content/general/7_integrations/grafana-integration/3_dashboards.md (hunk 3 above)
Comment (mayberryzane) [reply docs-content/general/7_integrations/grafana-integration/3_dashboards.md:35]: oops they were gitignored

File: docs-content/general/7_integrations/grafana-integration/2_setup.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+Once you have installed the highlight.io plugin, you can add a new data source and configure it with these fields:
+
+![Configuration](/images/docs/grafana/configuration.png)
+
+|Name|Description|
+|----|-----------|
+|`Highlight Backend URL`|The URL used to query highlight.io data. If you are using cloud-managed highlight.io, set this to `https://pri.highlight.io`.|
+|`OAuth Token URL`|The URL used to retrieve an OAuth access token. If you are using cloud-managed highlight.io, set this to `https://pri.highlight.io/oauth/token`.|
+|`Project ID`|The highlight.io project ID, accessible from the URL slug of your highlight.io project, e.g. `https://app.highlight.io/{project_id}/`. If you want to query data from multiple projects, you can set up multiple data sources. To query the demo project, set this to 1344.|
+|`Client ID`|If you are using cloud-managed highlight.io, set this to your OAuth client ID. If you are setting up the data source for the first time and need a client ID, reach out to us at [support@highlight.io](mailto:support@highlight.io). If you are self-hosting highlight.io or using the demo project, leave this blank.|
```
[the remark this answers is not in the document]
Comment (mayberryzane) [reply docs-content/general/7_integrations/grafana-integration/2_setup.md:33]: good catch, I don't think what I wrote is accurate - by default they would have to set up their oauth permissions.

[the remark this answers is not in the document]
Comment (mayberryzane) [reply docs-content/general/7_integrations/grafana-integration/2_setup.md]: added OAuth setup instructions in a section below

**Pull request merged**